src/sentry/static/sentry/app/views/issueList/actions.tsx

```diff
@@ -425,6 +430,23 @@ class IssueListActions extends React.Component<Props, State> {
             <Checkbox onChange={this.handleSelectAll} checked={pageSelected} />
           </ActionsCheckbox>
           <ActionSet>
+            <Feature organization={organization} features={['organizations:inbox']}>
+              <div className="btn-group hidden-md hidden-sm hidden-xs">
```

Comment:
Do we want to show this in the dropdown when it gets hidden?

![image](https://user-images.githubusercontent.com/79684/98619982-2fbad680-22b9-11eb-8e56-ab63fc8ed59b.png)

Reply:
oh good point